specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -636,7 +739,8 @@
         },
         "configurations": {
           "type": "object",
-          "description": "The cluster configurations."
+          "description": "The cluster configurations.",
```

Comment:
How does this object look like? Can you add schema for this too?

Reply:
> How does this object look like? Can you add schema for this too?

Usually, it is like `Dictionary<string, Dictionary<string, xx>>`, I can not add schema for this, because the xx maybe boolean or interger, backend depends on this to be flexible in some way.

This PR is in order to fix S360 and there is not related issue with this property, can I skip this comment?

Thanks for your comment.